source/_posts/fast_api_nayan_blog.md

```diff
@@ -0,0 +1,87 @@
+
```

Comment:
keyword should use in content also

Reply:
Added Machine Learning and Artificial Intelligence at the bottom. Please check

Reply:
Done